### ISSUE TITLE
Investigate failure on Firefox+Selenium2

### ISSUE THREAD

**Issue by uuf6429** [minkphp/webdriver-classic-driver#33]
Example case: https://github.com/minkphp/webdriver-classic-driver/actions/runs/11628711441/job/32384348916
```diff
---·Expected
+++·Actual
@@ @@
-'Dropped·small!'
+'Drop·here'
```

I noticed these failures in the past, but no idea when they started.

**Comment by aik099** [comment]
The Selenium Server 2.x on a specific Firefox version used in the Docker image isn't dragging elements properly. We've only found out about this issue after a https://github.com/minkphp/driver-testsuite/pull/95 test was added to a test suite.

This is no such problem on Selenium 3.x, because it uses a Docker image with a newer Firefox version.

To solve this we need to:
* either remove that test;
* or create a custom docker image with a newer Firefox version.

**Comment by uuf6429** [comment]
Thanks, I've created a request here: https://github.com/SeleniumHQ/docker-selenium/issues/2454

**Comment by aik099** [comment]
@uuf6429 , What are our next steps considering, that SeleniumHQ won't be creating a custom image for us?

I've understood the build instructions, but should build that custom image on every driver test run OR we can somehow build it and upload it somewhere, where we can use it later?

**Comment by uuf6429** [comment]
Let's continue the discussion in https://github.com/minkphp/MinkSelenium2Driver/issues/391, then we can update this driver's workflow when the time comes.

**Comment by uuf6429** [comment]
Closed by #49 / #391.
